versioned/storage/common/src/main/java/org/projectnessie/versioned/storage/common/persist/ObjId.java

```diff
@@ -110,20 +118,43 @@ public static ObjId objIdFromBytes(ByteString bytes) {
     }
   }
 
+  /**
+   * Creates an {@link ObjId} from its bytes representation, assuming that all remaining data in
```

Comment:
the `remaining` piece is a bit misleading... the byte array is treated whole, is it not? There is no moving pointer :thinking: